### ISSUE TITLE
Alternate Opensource geocoding API

### ISSUE THREAD

**Issue by Vishi13** [scalability4all/voice-enabled-chatbot#44]
**Is your feature request related to a problem? Please describe.**
To use google geolocation api, User has to  fill the credit details. Instead, we can use opensource geocoding api available.

**Describe the solution you'd like**
We can use alternate geocoding api available to get the same feature.

**Comment by satyammittal** [comment]
Yes, I am also in support of using open source APIs. You can raise a PR stating the change. 👍

**Comment by Vishi13** [comment]
Sure! I will start working on it :)
